p4-16/psa/examples/psa-example-recirculate.p4

```diff
@@ -102,6 +101,12 @@ parser IngressParserImpl(
 
     state parse_recirc_header {
         user_meta.recirc_header = recirc_meta;
+        // @hanw - You could write a P4 program like this, but then
+        // recirculated packets would have no headers parsed and valid
+        // during ingress processing, true?  It seems like a much
+        // better and more typical example if we use 'transition
+        // parse_ethernet' instead of 'transition accept' on the next
+        // line.  Do you agree?
```

Comment:
Is this still a question?

Reply:
I agree this state should transit to parse_ethernet.

Reply:
I think I know the answer, but would appreciate it if @hanw could look at this part and confirm my belief, and the suggested change to the example program.

Reply:
Thanks, Han.  I have changed the transition statement to parse_ethernet, and deleted the question about it in comments.